embassy-executor/src/arch/xtensa.rs

```diff
@@ -8,7 +8,6 @@ mod thread {
     use core::marker::PhantomData;
     use core::sync::atomic::{AtomicBool, Ordering};
 
-    #[cfg(feature = "nightly")]
     pub use embassy_macros::main_riscv as main;
```

Comment:
might wanna _not_ reexport the riscv macro?

Reply:
it was already broken before, will fix in a separate PR.